### ISSUE TITLE
Add develop command to frontend_addon

### ISSUE THREAD

**Issue by ichim-david** [plone/cookieplone-templates#74]
I propose we add a develop command here
https://github.com/plone/cookieplone-templates/blob/main/frontend_addon/%7B%7B%20cookiecutter.__folder_name%20%7D%7D/package.json#L15
"develop": "missdev --no-config  --fetch-https",

This allows me to develop any new add-ons I set in mrsdeveloper
ex:
{
  "core": {
    "output": "./",
    "package": "@plone/volto",
    "url": "git@github.com:plone/volto.git",
    "https": "https://github.com/plone/volto.git"
  },
  "volto-slate-glossary": {
    "output": "packages",
    "url": "git@github.com:rohberg/volto-slate-glossary.git",
    "https": "https://github.com/rohberg/volto-slate-glossary.git",
    "package": "@rohberg/volto-slate-glossary",
    "branch": "only-first-occurence---use-wrapped-view",
    "path": "src"
  }
}

Would be cool if we could make it easy with another command to add an add-on within volto-config addons key but for now I want to know what is the current feeling regarding this proposal.

**Comment by ichim-david** [comment]
@ericof @davisagli @sneridagh what do you guys think about my proposal? 
If you don't agree what would you propose instead if someone wants to have an easy add-on developing and registration in this new monorepo way?

**Comment by davisagli** [comment]
@ichim-david This is already included when you run `make install`:

https://github.com/plone/cookieplone-templates/blob/main/frontend_addon/%7B%7B%20cookiecutter.__folder_name%20%7D%7D/Makefile#L41

Is there a use case for running your develop command without running the other commands that are part of `make install`?

**Comment by davisagli** [comment]
@ichim-david I might be missing the point, but why would you want to check out things without installing them so that you can use them?

**Comment by ichim-david** [comment]
> @ichim-david I might be missing the point, but why would you want to check out things without installing them so that you can use them?

@davisagli I'm not saying that I want todo that but up to this point we have the documentation on how to add addons using mrs developer using make develop
https://6.docs.plone.org/volto/addons/index.html#use-mrs-developer-to-manage-the-development-cycle
This has now been basically merged into make install. We are sort of loosing that gained knowledge because now things are done differently.
Again not a major deal if it's documented as make install is now the equivalent of make develop and make install from previous system.

**Comment by ichim-david** [comment]
Closing as I got my feedback that make install does what I needed as part of the "develop" command that I wanted with the package.json scripts section.

**Comment by davisagli** [comment]
@ichim-david I think we've had different habits for how to do this. `make install` has included running missdev since the time I started using volto, so I always have just run `make install`.

(I think part of what happened is that yarn install used to run the `develop` command via a `preinstall` script, so it all happened automatically with configuration in package.json. But pnpm didn't support this feature, so we had to switch to a make command to do it all together.)
